examples/plot_deflect_spatial_filter.py

```diff
@@ -0,0 +1,236 @@
+"""
+=======================================
+Compute spatial filter based on DeFleCT
+=======================================
+
+Compute spatial filters obeying a combination of several constraints in the
+DeFleCT framework. Apply spatial filters to evoked data and plot SNR curves.
+Apply spatial filters to epoched data and compute between-labels spectral
+connectivity.
+
+The constraints that can be implemented are:
+- Suppress cross-talk within a small set of labels
+- Minimize cross-talk from other locations anywhere in the brain
+- Minimize the effect of noise (represented by noise covariance matrix).
+
+Different sensor types are combined by whitening with noise covariance matrix.
+First label is the target. Cross-talk with other labels should be zero. 
+Cross-talk with other vertices will be minimized.
+For reference, see http://www.ncbi.nlm.nih.gov/pubmed/23616402
```

Comment:
you're example is too long. The more concise the easier it is to reuse.

I would just stick to the MEG data.